pkg/mounter/fuse_containerized_mounter.go

```diff
@@ -201,6 +217,22 @@ func (mounter *ContainerizedFuseMounter) labelsAndListOptionsFor(target string)
 	return labels, listOptions
 }
 
+func (mounter *ContainerizedFuseMounter) ensureSeriveAccount(ctx context.Context) error {
```

Comment:
Do we really want this? Would this requires additional RBAC permission? maybe we should just add a service account into our installation yaml.

Reply:
additional RBAC policy is a problem, but should we create such a SA by default? i think most of the users will not use RRSA

Reply:
I think it is OK. This SA does not have any RBAC role binding.